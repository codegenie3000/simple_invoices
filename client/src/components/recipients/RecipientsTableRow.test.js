import React from 'react';
import { render, mount, shallow } from 'enzyme';

import RecipientTableRow from './RecipientsTableRow';

describe('<RecipientTableRow />', () => {
    it('renders given data', () => {
        const recipientData = {
            name: 'Jimmy McNulty',
            balance: 1000.25,
            id: 'abc123'
        };
        const component = shallow(
            <RecipientTableRow
                id={recipientData.id}
                name={recipientData.name}
                balance={recipientData.balance}
            />
        );
        expect(component).toMatchSnapshot();
    });
});