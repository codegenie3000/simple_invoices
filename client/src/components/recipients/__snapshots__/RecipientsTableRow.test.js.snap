// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RecipientTableRow /> renders given data 1`] = `
<tr
  className="tr"
>
  <td
    className="td"
  >
    <Link
      replace={false}
      to="api/recipients/abc123"
    >
      Jimmy McNulty
    </Link>
  </td>
  <td
    className="td"
  >
    1,000.25
  </td>
</tr>
`;
